recipe/meta.yaml

```diff
@@ -1,42 +1,41 @@
-{% set version = "0.8.7" %}
-{% set sha256 = "c69f55668758e6e18ab5754ed3e6f74faea81939594467717345f1ca62858b3d" %}
+{% set name = "ipympl" %}
+{% set version = "0.9.3" %}
+{% set sha256 = "49bab75c05673a6881d1aaec5d8ac81d4624f73d292d154c5fb7096f10236a2b" %}
 
 package:
-  name: ipympl
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: ipympl-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/i/ipympl/ipympl-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
   number: 0
-  # It's currently not supported on s390x. 
+  # It's currently not supported on s390x.
   # Yarn is unsatisfiable dependency on s390x.
-  skip: True  # [s390x]
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: True  # [s390x or py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -v
```

Comment:
`-vv`?

Reply:
`-vv` is useless from my experience. `-v` is enough to get the build logs. More `v`s just makes pip output debug logs.